x-pack/plugins/infra/public/components/asset_details/types.ts

```diff
@@ -38,6 +41,10 @@ export enum FlyoutTabIds {
 export type TabIds = `${FlyoutTabIds}`;
 
 export interface TabState {
+  overview?: {
+    dateRange: StringDateRange;
```

Comment:
This will change once we have the date picker for now the consumer of the component will set the time range